jekyll/_cci2/env-vars.md

```diff
@@ -517,8 +514,11 @@ Variable                    | Type    | Value
 `CIRCLE_USERNAME`           | String  | The GitHub or Bitbucket username of the user who committed the build.
```

Comment:
Committed the build probably needs changing...

Reply:
"whose commit triggered the pipeline" perhaps?

Reply:
For webhooks it's the pusher rather than committer, for API calls it's the API caller. So maybe "the user who triggered the pipeline"?

Reply:
Yeah that's what I was thinking... hopefully Server people won't be confused by pipeline

Reply:
oh, so this isn't the same user that's attributed for credit usage?

Reply:
when you say the user who triggered the pipeline, does that’s  mean for pipelines triggered by API it will be the user whose token I am using?

also if i do "rerun" or "rerun from failed" on a build started by `userA` does the `CIRCLE_USERNAME` change to the person who re-ran?